lib/pushmi_pullyu/aip.rb

```diff
@@ -0,0 +1,40 @@
+require 'pushmi_pullyu'
+require 'bagit'
+require 'archive/tar/minitar'
+
+module PushmiPullyu::AIP
+  # Exceptions
+  class BagInvalid < StandardError; end
+  class NoContentFilename < StandardError; end
+  class FedoraFetchError < StandardError; end
+  class SolrFetchError < StandardError; end
+
+  def self.create(noid, should_skip_download: false, should_clean_work_directories: true)
+    # Return the filename of the created AIP tarball
+    PushmiPullyu::AIP::Creator
+      .new(noid)
+      .run(should_skip_download: should_skip_download,
+           should_clean_work_directories: should_clean_work_directories)
+    aip_filename(noid)
```

Comment:
Your `PushmiPullyu::AIP::Creator.new(...).run(....)` returns a `aip_filename` Why not just use this one? Or is this `aip_filename(noid)` different some how?